docs/Fuzzing.md

```diff
@@ -0,0 +1,32 @@
+# Fuzz Testing in eBPF-For-Windows
+
+## Overview
+Fuzz testing is a test methodology that finds a class of bugs in the code-base
+by generating random inputs and verifying that the code doesn't crash.
+
+## Tests
+The fuzzing tests are in the repo under tests/fuzz. Fuzz tests execute as part
+of each CI/CD workflow. The test generates a random block of bytes with a
```

Comment:
Line 8 has "tests" plural (twice).  Lines 9, 12, 30, 32 have "The test" implying singular.
Be consistent.